tests/pending/pos-custom-args/captures/immutable-cycle.scala

```diff
@@ -0,0 +1,15 @@
+import caps.*
+
+class Node(val payload: Int) extends Mutable:
+  var next: Node = ???
```

Comment:
Would this mean that `Node` has a read-only reference to `next`? (even before the immutable scope)